docs/core/deploying/trim-warnings/il2058.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "IL2058: Parameters passed to 'Assembly.CreateInstance' cannot be statically analyzed"
+description: "Learn about trim warning IL2058: AssemblyCreateInstanceParameters"
+ms.date: 07/23/2021
+ms.topic: reference
+author: mateoatr
+ms.author: matorre
+f1_keywords:
+  - "IL2058"
+---
+# IL2058: Parameters passed to 'Assembly.CreateInstance' cannot be statically analyzed'
+
+## Cause
+
+A call to <xref:Assembly.CreateInstance> was made in the context of trimming.
```

Comment:
"in the context of trimming" - not sure what that means honestly. Maybe just "A call to Assembly.CreateInstance was found in the analyzed code."